bears-app/README.md

```diff
@@ -1,70 +1,245 @@
+## Coding Standards
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+We use a combination of linting and best practices to guide code consistency.
 
-### `npm run eject`
+<br>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Our linters include
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+[![js-standard-style](https://img.shields.io/badge/code%20style%20JS-standard-brightgreen.svg)](http://standardjs.com)
+[![react/recommended](https://img.shields.io/badge/code%20style%20React-recommended-brightgreen.svg)](https://github.com/yannickcr/eslint-plugin-react)
+[![prettier/recommended](https://img.shields.io/badge/code%20format/prettier-%20recomended-brightgreen.svg)](https://github.com/prettier/eslint-plugin-prettier)
+[![stylelint](https://img.shields.io/badge/code%20format/stylelint-%20standard-brightgreen.svg)](https://github.com/cypress-io/eslint-plugin-cypress)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<br>
 
-## Learn More
+#### Linting and Formatting Scripts
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+We include a few scripts in `package.json` that can be executed to interact with our standards.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Run eslint on `.js` and `.jsx` files
 
-### Code Splitting
+```shell
+npm run lint:js
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Auto-fix lint errors
+```shell
+npm run lint:js:fix
+```
 
-### Analyzing the Bundle Size
+Format `.js` and `.jsx` files with prettier.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+```shell
+npm run format
+```
 
-### Making a Progressive Web App
+Run stylelint on `.scss` files
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+```shell
+npm run lint:scss
+```
```

Comment:
npm run lint:scss 
> bears-app@0.1.0 lint:scss 
> stylelint 'src/**/*.scss' --syntax scss 
Invalid option "--syntax". Did you mean "--config"?

Reply:
updated